src/views/splash/custom.jsx

```diff
@@ -0,0 +1,111 @@
+var React = require('react');
+
+var Box = require('../../components/box/box.jsx');
+var Carousel = require('../../components/carousel/carousel.jsx');
+
+require('./splash.scss');
+
+var CustomRows = React.createClass({
+    type: 'CustomRows',
+    getDefaultProps: function () {
+        return {
+            featured: {}
+        };
+    },
+    rowMap: {
+        custom_projects_by_following: {
+            id: 'splash.projectsByScratchersFollowing',
+            defaultMessage: 'Projects by Scratchers I\'m Following'
+        },
+        custom_projects_loved_by_following: {
+            id: 'splash.projectsLovedByScratchersFollowing',
+            defaultMessage: 'Projects Loved by Scratchers I\'m Following'
+        },
+        custom_projects_in_studios_following: {
+            id: 'splash.projectsInStudiosFollowing',
+            defaultMessage: 'Projects in Studios I\'m Following'
+        }
+    },
+    componentWillReceiveProps: function (){
+        var formatMessage = this.props.intl.formatMessage;
```

Comment:
I agree... it doesn't look like this does anything.  This kind of stuff shouldn't happen in `componentWillReceiveProps` either — seems like it was supposed to go into `render` but didn't end up there, and then it looks like it is `render` so it didn't get cleaned up? Maybe?